server/api/eventinfo/eventinfo.controller.js

```diff
@@ -59,6 +63,33 @@ function removeEntity(res) {
   };
 }
 
+function saveEventInfoImage(files, fields){
+  var imagePaths = [],
+      asyncTasks = [];
+  if (!files){return imagePaths;}
+  files.files.forEach(function(file) {
+    var name = Date.now().toString()+ '_' + file.path.substring(file.path.lastIndexOf('/')).substring(1);
+    var path = config.imageUploadPath  + 'eventInfoImages';
```

Comment:
node should have a utility for getting the filename, probably like `path.basename`